compose/compiler/compiler-hosted/src/main/java/androidx/compose/compiler/plugins/kotlin/lower/decoys/CreateDecoysTransformer.kt

```diff
@@ -150,6 +159,64 @@ class CreateDecoysTransformer(
         }
     }
 
+    override fun visitCall(expression: IrCall): IrExpression {
+        val function = (expression.symbol.owner as? IrSimpleFunction)
+            ?: return super.visitCall(expression)
+        val property = (function.correspondingPropertySymbol?.owner)
+            ?: return super.visitCall(expression)
+
+        if (function.isDecoy() || !function.shouldBeRemapped()) {
```

Comment:
Should we also skip this part for k/js (here and in other cases below)? 

`if (function.isDecoy() || !function.shouldBeRemapped() || context.platform.isJs()) {` ?